CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+## 0.9.1
+
+* The Broccoli build system was replaced by the Gulp build system
+* Complete replacement of the Backbone models with a Flux structure.
+  The components retrieve their data from stores via events on place.
+  Actions now fetch data from the Marathon API through an replaceable
+  Ajax-wrapper.
+
+## 0.9.0
+
+* Marathon UI lives now in it's own Git repository
+* Update to React 0.13
+* Testing via Mocha
```

Comment:
Very good point. @aldipower could you rewrite this to be inline with Marathon's [changelog](https://github.com/mesosphere/marathon/blob/master/changelog.md)